pkgs/development/python-modules/nix-kernel/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pexpect
+, notebook
+, nix
+}:
+
+buildPythonPackage rec {
+  pname = "nix-kernel";
+  version = "0.1";
```

no official releases yet
```suggestion
  version = "unstable-2020-04-26";
```

---

done.